apps/src/lib/client/tx.rs

```diff
@@ -420,6 +426,81 @@ pub async fn submit_transfer(ctx: Context, args: args::TxTransfer) {
     process_tx(ctx, &args.tx, tx, &keypair).await;
 }
 
+pub async fn submit_create_nft(ctx: Context, args: args::NftCreate) {
+    let file = File::open(&args.nft_data).expect("File must exist.");
+    let nft: Nft = serde_json::from_reader(file)
+        .expect("Couldn't deserialize nft data file");
+
+    let vp_code = match &nft.vp_path {
+        Some(path) => {
+            std::fs::read(path).expect("Expected a file at given code path")
+        }
+        None => ctx.read_wasm(VP_NFT),
+    };
+
```

Comment:
It's a bit of a hack, but it's a nicer UX - we can default the signer to the NFT creator with:
```suggestion
    let signer = Some(WalletAddress::new( nft.creator.to_string()));
```
(1/2)